range-slider/src/app/app.component.ts

```diff
@@ -11,8 +12,20 @@ export class AppComponent {
   public someMin: number = -10;
   public someMax: number = 10;
 
+  @ViewChild('slider') slider: NouisliderComponent;
+
   onChange(value: any) {
     console.log('Value changed to', value);
+    this.someValue = [value[0], this.someValue[1]];
+  }
```

Comment:
You could do that absolutely (this.someValue[0]=value[0]). What I did was not change too much your code. As I said there are many ways of doing it. We should discuss for a more idiomatic solution. As for the array, I think is something that nouislider needs. If the component expects that datatype one should use that.